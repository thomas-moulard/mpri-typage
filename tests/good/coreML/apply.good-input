val apply f x = f x
