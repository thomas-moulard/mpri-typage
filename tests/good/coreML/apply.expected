apply = fun f.fun x.f x

apply : forall 'a 'b.(('b -> 'a) -> ('b -> 'a))
